### PR TITLE
[VENTUS][README] Add pocl example and improve readability.

### PR DESCRIPTION
**Pull request by xlinsist**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,38 +65,52 @@ NOTE: OpenCL host side program should be linked with icd loader `-lOpenCL`.
...
 0:
   Vendor:   THU
     Name:   Ventus GPGPU device
  Version:   2.2 HSTR: THU-ventus-gpgpu
 ```
 
-Also, you can try to set `POCL_DEBUG=all` and run example under `<pocl-build-dir>` to see the full OpenCL software stack execution pipeline. For example(Work in progress).
+Also, you can try to set `POCL_DEBUG=all` and run example under `<pocl-build-dir>` to see the full OpenCL software stack execution pipeline. For example:
+```
+$ ./<pocl-install-dir>/examples/example1
```
Comment (zhoujingya) [inline README.md:78]: ./<pocl-install-dir>/examples/example1/exapmple1

**Comment by zhoujingya** [comment]
Maybe title can be changed to `[VENTUS][readme] Add pocl example`?@xlinsist

File: .gitignore [hunk 2]
```diff
@@ -30,6 +30,9 @@
 /build*/
 /install*
 
+vecadd*
```
Comment (Jules-Kong) [inline .gitignore:33]: What is the purpose of adding these?

File: README.md [hunk 3]
```diff
@@ -65,38 +65,52 @@ NOTE: OpenCL host side program should be linked with icd loader `-lOpenCL`.
...
+// The following output should be shown:
 LIST OF DEVICES:
 0:
   Vendor:   THU
     Name:   Ventus GPGPU device
  Version:   2.2 HSTR: THU-ventus-gpgpu
 ```
 
-Also, you can try to set `POCL_DEBUG=all` and run example under `<pocl-build-dir>` to see the full OpenCL software stack execution pipeline. For example(Work in progress).
+Also, you can try to set `POCL_DEBUG=all` and run example under `<pocl-build-dir>` to see the full OpenCL software stack execution pipeline. For example:
```
Comment (Jules-Kong) [inline README.md:76]: This example may not be appropriate. The comparison before and after the environment variable is imported is:    
Screenshot from 2024-09-26 15-11-25

File: README.md [hunk 4]
```diff
@@ -65,38 +65,52 @@ NOTE: OpenCL host side program should be linked with icd loader `-lOpenCL`.
...
 
 
 ### 4: Compiler using example
 
-we can now use our built compiler to generate an ELF file, and using [spike](https://github.com/THU-DSP-LAB/ventus-gpgpu-isa-simulator) to complete the isa simulation.
+We can now use our built compiler to generate an ELF file, and using [spike](https://github.com/THU-DSP-LAB/ventus-gpgpu-isa-simulator) to complete the isa simulation.
 
 > Cause the address space requirement in spike, we use a customized linker script for our compiler
 
-Take `vecadd.cl` below as an example :
```
Comment (Jules-Kong) [inline README.md:84]: "Take the following `vecadd.cl` as an example and place it under `<path-to-llvm-ventus>`."
Maybe this is more appropriate, and then just change line 99?

**Comment by Jules-Kong** [comment]
It is recommended that the title of commit id 43ae5502 be in the following format: "[VENTUS][readme] Supplement pocl example".

File: README.md (hunk 1 above)
Comment (xlinsist) [reply README.md]: Thanks for pointing it out!

File: .gitignore (hunk 2 above)
Comment (xlinsist) [reply .gitignore]: Fixed, thanks!

**Comment by xlinsist** [comment]
> Maybe title can be changed to `[VENTUS][readme] Add pocl example`?@xlinsist

> It is recommended to merge two commits into one, such as "[VENTUS][NFC] Title..."

To ensure consistency, this PR will be prefixed with `[VENTUS]`, and one PR will include one commit.

File: README.md [hunk 5]
```diff
@@ -51,7 +51,7 @@ You can dive into `build-ventus.sh` file to see the detailed information about b
 
 ### 3: Bridge icd loader
 
-Run `export VENTUS_INSTALL_PREFIX=<path_to_install>` to set `VENTUS_INSTALL_PREFIX` environment variable(system environment variable recommended)
+Run `export VENTUS_INSTALL_PREFIX=<path_to_install>` to set `VENTUS_INSTALL_PREFIX` environment variable(system environment variable recommended). If you execute `build-ventus.sh` in default setting, `<path_to_install>` should be `<path-to-llvm-ventus>`/install.
```
[the remark this answers is not in the document]
Comment (xlinsist) [reply README.md]: Got it. Now I have:
1. replaced `<llvm-ventus-parentFolder>` with `<path-to-llvm-ventus>/..`.
2. change `<path_to_install>` to `<path-to-llvm-ventus>/../install`, and correct the original description.

File: README.md (hunk 3 above)
Comment (xlinsist) [reply README.md:74]: Thanks for pointing it out! Now I change to use `vecadd`, the same example given in "4.4: Driver using example" and correct the statement.

File: README.md (hunk 4 above)
Comment (xlinsist) [reply README.md:84]: Sure, now I rephrase it as "First, name the following program `vecadd.cl`, and place it under `<path-to-llvm-ventus>`".

**Comment by xlinsist** [comment]
Thank you for the feedback! All the changes are now complete.

**Review by zhoujingya — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -42,62 +42,62 @@ The following packages are needed for other repositories:
...
+* `export POCL_DIR=<path-to-pocl-dir>`, default folder path will be set to be `<path-to-llvm-ventus>/../pocl`.
+* `export OCL_ICD_DIR=<path-to-ocl-icd-dir>`, default folder path will be set to be `<path-to-llvm-ventus>/../ocl-icd`.
 
-You can dive into `build-ventus.sh` file to see the detailed information about build process
+You can dive into `build-ventus.sh` file to see the detailed information about build process.
 
 ### 3: Bridge icd loader
 
-Run `export VENTUS_INSTALL_PREFIX=<path_to_install>` to set `VENTUS_INSTALL_PREFIX` environment variable(system environment variable recommended)
+Run `export VENTUS_INSTALL_PREFIX=<path_to_install>` to set `VENTUS_INSTALL_PREFIX` environment variable(system environment variable recommended), default folder path will be set to be `<path-to-llvm-ventus>/../install`.
```
Comment (Jules-Kong) [inline README.md:54]: Typically, this is set to `<path-to-llvm-ventus>/install`.

Comment (xlinsist) [reply README.md]: Thanks! I have fixed it.

**Comment by Jules-Kong** [comment]
Some suggestions:   
- The test cases used by the user and the generated files of the test cases do not need to be put in the public repository.
- Try to modify only the suggested parts each time to be more friendly to committers. If you suddenly have new code that is a new function or feature, you can submit another PR.    
@xlinsist  Thank you very much for your contribution.  :）

**Comment by xlinsist** [comment]
I have modified the commit as requested. Thanks!

**Review by Jules-Kong — APPROVED** [review]
lgtm

**Pull request merged**